manifests/integreatly-3scale/0.10.0-mas/3scale-operator.v0.10.0-mas.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1019 @@
+            - apiGroups:
+                - integreatly.org
+              resources:
+                - grafanadashboards
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - update
+                - watch
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - podmonitors
+                - prometheusrules
+                - servicemonitors
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - update
+                - watch
+            - apiGroups:
+                - policy
+              resources:
+                - poddisruptionbudgets
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - rolebindings
+                - roles
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes/custom-host
+              verbs:
+                - create
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes/status
+              verbs:
+                - get
+          serviceAccountName: 3scale-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - 3scale
+    - API
+  links:
+    - name: GitHub
+      url: https://github.com/3scale/3scale-operator
+    - name: Documentation
+      url: https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.12/html/installing_3scale/install-threescale-on-openshift-guide#deploying-threescale-using-the-operator
+  maintainers:
+    - email: msoriano+3scaleoperator@redhat.com
+      name: Miguel Soriano
+    - email: eastizle+3scaleoperator@redhat.com
+      name: Eguzki Astiz
+  maturity: stable
+  provider:
+    name: Red Hat
+  relatedImages:
+    - image: registry.redhat.io/3scale-mas/3scale-rhel7-operator@sha256:06c81544f81f8d7e4e4c5d44c9ab9f4109f1eadcc24b021ce031a853f0f9e2d3
```

Comment:
Will images be pullable to test this onebefore a release is done?

Reply:
I think that all the images that need to be pulled are mirror mapped in the env block from L397.